python_src/src/lamp_py/performance_manager/flat_file.py

```diff
@@ -0,0 +1,192 @@
+import datetime
+import os
+from typing import List
+
+import sqlalchemy as sa
+import pyarrow
+
+from lamp_py.aws.s3 import write_parquet_file
+from lamp_py.performance_manager.gtfs_utils import (
+    get_service_date_from_timestamp,
+    static_version_key_from_service_date,
+)
+from lamp_py.postgres.postgres_schema import (
+    VehicleEvents,
+    VehicleTrips,
+    StaticStopTimes,
+    StaticStops,
+)
+from lamp_py.postgres.postgres_utils import DatabaseManager
+
+
+def write_flat_files(
+    db_manager: DatabaseManager, dates: List[datetime.datetime]
+) -> None:
+    """write flat files to s3 for datetimes"""
+    dates_to_write = {get_service_date_from_timestamp(d) for d in dates}
+
```

Comment:
I think there is a chance that the service_date pulled from this function won't match the service_date's processed during the event loop. But I'm not totally sure on the edge cases that could cause issues here. 

Might be worth talking through the different variations of possible service_dates's returned compared to service_date's processed based on timezone and time of day variations.